modules/kmm-using-intree-modules.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-using-intree-modules_{context}"]
+
+= Using in-tree modules
+
+In some cases, you may need to configure the KMM Module to avoid loading an out-of-tree kernel module and instead use the in-tree module, running only the device plugin. In such cases, you can omit `moduleLoader` from the `Module` custom resource, and leave only the `devicePlugin` section, as shown in the following example.
```

Comment:
```suggestion
In some cases, you might need to configure the KMM Module to avoid loading an out-of-tree kernel module and instead use the in-tree module, running only the device plugin. In such cases, you can omit the `moduleLoader` parameter from the `Module` custom resource, and leave only the `devicePlugin` section, as shown in the following example.
```
- May vs. might: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may
- Command parameter guidelines: https://www.ibm.com/docs/en/ibm-style?topic=elements-commands#commands-parameters-and-options-in-general-text

Reply:
Made these changes.